Message.cpp

```diff
@@ -3,6 +3,7 @@
 #define DASH_MSG_PERIOD 100UL
 #define BMS_MSG_PERIOD 1000UL
 #define MC_SINGLE_REQUEST_PERIOD 100UL
+#define MC_PERMANENT_REQUEST_PERIOD 100UL
```

Comment:
@jatron FYI I realized constants default to 16bit in AVR so this forces ms timestamps to be unsigned longs, which are 32 bit. This might not actually be necessary, but better safe than sorry.